kratos/mpi/python/add_mpi_debug_utilities_to_python.cpp

```diff
@@ -18,8 +18,7 @@
 // Project includes
 #include "mpi/utilities/debug_utilities.h"
 
-namespace Kratos {
-namespace Python {
+namespace Kratos::Python {
```

Comment:
Hey, I did not know that could be used!!!

Reply:
Noob.